XMLHttpRequest/responsetext-decoding.htm

```diff
@@ -1,47 +1,89 @@
 <!doctype html>
 <html>
   <head>
+    <meta charset="utf-8">
     <title>XMLHttpRequest: responseText decoding</title>
     <script src="/resources/testharness.js"></script>
     <script src="/resources/testharnessreport.js"></script>
     <link rel="help" href="https://xhr.spec.whatwg.org/#the-responsetext-attribute" data-tested-assertations="following::ol[1]/li[4]" />
     <link rel="help" href="https://xhr.spec.whatwg.org/#text-response-entity-body" data-tested-assertations="following::ol[1]/li[2] following::ol[1]/li[3] following::ol[1]/li[4] following::ol[1]/li[5]" />
   </head>
   <body>
-    <div id="log"></div>
     <script>
+      function html(content) {
```

Comment:
how about adding create_ or build_ to the function name?

Reply:
Done.